### ISSUE TITLE
Wonderful Enchantments not compatible with modded enchanting tables

### ISSUE THREAD

**Issue by J4bb4r** [Majrusz/MinecraftMods#21]
I've disabled several enchantments from Wonderful Enchantments. They do not appear in chests, neither are items from treasure chests enchanted with the Wonderful Enchantments enchantments. I've checked in the vanilla enchanting table and none of the enchantments from Wonderful Enchantments appear, so everything is working as intended in that aspect. 

Problems seem to arise when you use any type of modded enchanting table, like the matrix enchanting table from Quark Oddities or the soul enchanter from Eidolon. Whenever I try enchanting something with the matrix enchanting table or the soul enchanter, the enchantments from Wonderful Enchantments that I've disabled are available. 

If I were to specify which enchantments made me notice the issue it'd be Smelter and Vengeance of the Pufferfish - in case it's just those two in specific.

One thing worth mentioning is that in the newer versions of Quark and Quark Oddities, instead of just replacing the functionality of the enchanting table, the creator just made it so that when you place down an enchanting table it immidietly gets converted into a matrix enchanting table, so it doesnt work like a normal enchantint table. That, combined with the fact that the same thing happens with the soul enchanter from Eidolon makes me suspect that it's only an issue with modded enchanting tables.

**TLDR**: Disabling enchantments in the config doesn't affect modded enchanting tables, so they still appear in those regardless of whether you disabled them or not.

Here are links to the mods I mentioned:
- Eidolon: https://www.curseforge.com/minecraft/mc-mods/eidolon
- Quark: https://www.curseforge.com/minecraft/mc-mods/quark
- Quark Oddities: https://www.curseforge.com/minecraft/mc-mods/quark-oddities

**Comment by Majrusz** [comment]
Thanks for reporting this. In the next update 1.1.0 config for enchantments will work properly.
